renku/version.py

```diff
@@ -24,7 +24,7 @@
 except ImportError:
     from importlib_metadata import distribution  # type: ignore
 
-__version__ = "0.0.0"
+__version__ = "1.4.0"
```

Comment:
Can we keep this? I've checked `poetry-dynamic-versioning` works with it (although they recommend to use 0.0.0). I use Renku outside a venv and always have to set this.